src/chainstate/stacks/db/transactions.rs

```diff
@@ -426,41 +444,56 @@ impl StacksChainState {
                     Ok((ast, analysis)) => (ast, analysis),
                     Err(e) => {
                         // this analysis isn't free -- convert to runtime error
-                        debug!("Runtime error in contract analysis for {:?}: {:?}", &contract_id, &e);
-
+                        error!("Runtime error in contract analysis for {}: {}", &contract_id, &e);
+                        let receipt = StacksTransactionReceipt {
+                            transaction: tx.clone(),
+                            events: vec![],
+                            result: Value::err_uint(0),
```

Comment:
I'm not understanding the `result` field -- no code is running here, nor is there code running in the `TokenTransfer` branch.  If you don't need the `result` field to be anything meaningful, why not make `result` into an `Option`?  Or, turn `StacksTransactionReceipt` into an `enum` of payload-specific variants?

Reply:
Like, is `(err 0)` a well-known, protocol-defined error code that's specific to a failed analysis?

Reply:
Addressed with https://github.com/blockstack/stacks-blockchain/pull/1319/commits/47e7de6066e76b2eced519d81dde8c2d52a6c742